#! /bin/sh /usr/share/dpatch/dpatch-run
## 551709_fix_info_link_entry.dpatch by Daniel Leidert (dale) <daniel.leidert@wgdd.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix 'gpg' link which should be 'gnupg1'.
## DP:
## DP: <URL:http://bugs.debian.org/551709>

@DPATCH@
diff -urNad trunk~/doc/gnupg1.info trunk/doc/gnupg1.info
--- trunk~/doc/gnupg1.info	2009-09-02 15:50:59.000000000 +0200
+++ trunk/doc/gnupg1.info	2010-01-22 02:49:42.916068194 +0100
@@ -13,7 +13,7 @@
 
 INFO-DIR-SECTION GNU Utilities
 START-INFO-DIR-ENTRY
-* gpg: (gpg).            OpenPGP encryption and signing tool (v1).
+* gpg: (gnupg1).            OpenPGP encryption and signing tool (v1).
 END-INFO-DIR-ENTRY
 
 
diff -urNad trunk~/doc/gnupg1.texi trunk/doc/gnupg1.texi
--- trunk~/doc/gnupg1.texi	2008-12-11 17:39:51.000000000 +0100
+++ trunk/doc/gnupg1.texi	2010-01-22 02:49:22.005041865 +0100
@@ -48,7 +48,7 @@
 
 @dircategory GNU Utilities
 @direntry
-* gpg: (gpg).            OpenPGP encryption and signing tool (v1).
+* gpg: (gnupg1).            OpenPGP encryption and signing tool (v1).
 @end direntry
 
 
